apps/ledger-live-mobile/src/components/Web3AppWebview/WalletAPIWebview.tsx

```diff
@@ -53,6 +54,8 @@ export const WalletAPIWebview = forwardRef<WebviewAPI, WebviewProps>(
 
     const internalAppIds = useInternalAppIds() || INTERNAL_APP_IDS;
 
+    useSwapHeaderNavigation(webviewRef);
+
```

Comment:
Why do we need to add this swap specific code in the common WalletAPIWebview code ?
Especially because you have access to the ref above in the PTX specific webview code